packages/gatsby/src/utils/webpack.config.js

```diff
@@ -16,6 +17,8 @@ const apiRunnerNode = require(`./api-runner-node`)
 const createUtils = require(`./webpack-utils`)
 const hasLocalEslint = require(`./local-eslint-config-finder`)
 
+const FRAMEWORK_BUNDLES = [`react`, `react-dom`, `scheduler`, `prop-types`]
```

Comment:
prop-types shouldn't exist in production. Does this need to be here?

Reply:
That's another thing, we do not remove propTypes completely when it's part of our node_modules I believe. This is not part of this PR. Thanks for pointing that out.